protoc-gen-connect-kotlin/src/main/kotlin/build/buf/protocgen/connect/Generator.kt

```diff
@@ -396,6 +396,18 @@ class Generator : CodeGenerator {
         }
         return ClassName(packageName, names.first())
     }
+
+    internal fun String.sanitizeKdoc(): String {
+        return this
+            // Remove trailing whitespace on each line.
+            .replace("[^\\S\n]+\n".toRegex(), "\n")
+            .replace("\\s+$".toRegex(), "")
+            .replace("\\*/".toRegex(), "&#42;/")
+            .replace("/\\*".toRegex(), "/&#42;")
+            .replace("""[""", """\[""")
+            .replace("""]""", """\]""")
```

Comment:
Are we sure about these two? When I look at the generated source in IDEA, this is how the doc is rendered:

![image](https://user-images.githubusercontent.com/398111/234628532-f136449f-338c-4d43-b3af-58946eea5d06.png)

Reply:
I think we may want to do:

```
.replace("""[""", "&#91;")
.replace("""]""", "&#93;")
```

instead.